lib/apytypes/_array_functions.py

```diff
@@ -851,19 +851,20 @@ def arange(
     frac_bits=None,
     bits=None,
     exp_bits=None,
-    mantissa_bits=None,
+    man_bits=None,
+    bias=None,
 ):
     """
-    Simple and naive version of arange.
+    Create an array with evenly spaced values within a given interval.
```

Comment:
* `arange(stop)`: Values are generated within the half-open interval [0, stop) (in other words, the interval including start but excluding stop).
*  `arange(start, stop)`: Values are generated within the half-open interval [start, stop).
*  `arange(start, stop, step)`: Values are generated within the half-open interval [start, stop), with spacing between values given by step.